### DISCUSSION TITLE
Dear Mr. mashb1t, regarding whether contronet xinsir6promax will be supported by fooocus

### DISCUSSION THREAD

**Discussion by BNP1111** [lllyasviel/Fooocus#3311] [General]
First of all thank you for your great work, 
xinsir6 the latest cnet promax model takes sdxl to a great level and due to the infinite possibilities created, 
I take the liberty of asking if you have any intention of integrating the xinsir cnet into the fooocus or if it exists on your list of plans, thanks again!

**Comment by mashb1t** [comment]
https://github.com/lllyasviel/Fooocus/discussions/3264#discussioncomment-10092349
